string kind
string cls_name
int64  obj_id
string value